prow/images/gardener-rotate/Dockerfile

```
@@ -1,15 +1,14 @@
-FROM eu.gcr.io/kyma-project/external/golang:1.18.3-alpine3.16
-
-# Commit details
-ARG commit
-ENV IMAGE_COMMIT=$commit
-LABEL io.kyma-project.test-infra.commit=$commit
-
-# Default to UTF-8 file.encoding
-ENV LANG C.UTF-8
+FROM golang:1.19.1-alpine3.16 AS builder
 
 WORKDIR /go/src/github.com/kyma-project/test-infra
 COPY . .
 
-RUN  CGO_ENABLED=0 go build -o /gardener-rotate/gardener-rotate -ldflags="-s -w" development/gardener-rotate/main.go
+RUN  CGO_ENABLED=0 go build -o /gardener-rotate -ldflags="-s -w" development/gardener-rotate/main.go
+
+
+FROM alpine:3.15.0
```

why not latest alpine? 
```suggestion
FROM alpine:3.16.2
```